cou/utils/__init__.py

```diff
@@ -15,9 +15,23 @@
 """Utilities for charmed-openstack-upgrader."""
 
 import os
+from logging import Logger
 from pathlib import Path
+from typing import Any
 
 from halo import Halo
 
 COU_DATA = Path(f"/home/{os.getenv('USER')}/.local/share/cou") if os.getenv("USER") else Path(".")
 progress_indicator = Halo(spinner="line", placement="right")
+
+
+def print_and_debug(message: Any, logger: Logger) -> None:
```

Comment:
not-blocking: We can use `inspect.stack()` to get info about file, which is calling this function, and use it for logger definition. Like this, logger will not need to be passed as argument.